src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/EditCaseListingAfterAdjournmentHandler.java

```diff
@@ -49,7 +45,8 @@ public ServiceDataResponse<ServiceData> handle(ServiceData serviceData) {
 
         String caseId = getCaseReference(serviceData);
 
-        StartEventResponse startEventResponse = coreCaseDataService.startCaseEvent(LIST_CASE, caseId, CASE_TYPE_ASYLUM);
+        StartEventResponse startEventResponse = coreCaseDataService
+            .startCaseEvent(EDIT_CASE_LISTING, caseId, CASE_TYPE_ASYLUM);
         AsylumCase asylumCase = coreCaseDataService.getCaseFromStartedEvent(startEventResponse);
 
         updateListCaseHearingDetails(serviceData, asylumCase);
```

Comment:
it seems that the asylumCase is being updated in updateListCaseHearingDetails method so I believe it should be passed back and used in triggerSubmitEvent?

Reply:
Yeah, I could do that but it is not really necessary since 'asylumCase' is a reference variable and so passes it's value 'memory address' to the function.